packages/ui/src/components/Designer/index.tsx

```diff
@@ -219,59 +226,85 @@ Check this document: https://pdfme.com/docs/custom-schemas`);
 
   return (
     <Root size={size} scale={scale}>
-      <CtlBar
-        size={sizeExcSidebar}
-        pageCursor={pageCursor}
-        pageNum={schemasList.length}
-        setPageCursor={(p) => {
-          if (!mainRef.current) return;
-          mainRef.current.scrollTop = getPagesScrollTopByIndex(pageSizes, p, scale);
-          setPageCursor(p);
-          onEditEnd();
-        }}
-        zoomLevel={zoomLevel}
-        setZoomLevel={setZoomLevel}
-        {...pageManipulation}
-      />
-      <Sidebar
-        hoveringSchemaId={hoveringSchemaId}
-        onChangeHoveringSchemaId={onChangeHoveringSchemaId}
-        height={mainRef.current ? mainRef.current.clientHeight : 0}
-        size={size}
-        pageSize={pageSizes[pageCursor] ?? []}
-        activeElements={activeElements}
-        schemas={schemasList[pageCursor] ?? []}
-        changeSchemas={changeSchemas}
-        onSortEnd={onSortEnd}
-        onEdit={(id: string) => {
-          const editingElem = document.getElementById(id);
-          editingElem && onEdit([editingElem]);
+      <DndContext
+        onDragEnd={(event) => {
+          // Triggered after a scheme is dragged & dropped from the left sidebar.
+          if (!event.active) return;
+          const active = event.active;
+
+          const rect = paperRefs.current[pageCursor].getBoundingClientRect();
+          const initialTop = (active.rect.current.initial?.top || 0) - rect.top;
+          const initialLeft = (active.rect.current.initial?.left || 0) - rect.left;
+          const _scale = scale < 1 ? scale + 1 : scale;
+          const adjust = 0.915; // TODO: Investigate later as to why it needs to be adjusted.
```

Comment:
When dragging an element to the bottom of the page, its position gets misaligned.

Furthermore, if the page is zoomed in, the misalignment becomes even greater.

However, I have determined that this issue does not qualify as a release blocker, and we can fix it in the future.